modules/ai/assets/js/editor/page-content.js

```diff
@@ -144,7 +144,15 @@ const PageContent = (
 								additionalOptions={ additionalOptions }
 								credits={ credits }
 								usagePercentage={ usagePercentage }
-							/>
+							>
+								<UsageMessages
+									showBanner={ showBanner }
+									markBannerAsViewed={ markBannerAsViewed }
+									hasSubscription={ hasSubscription }
+									usagePercentage={ usagePercentage }
+									sx={ { mb: 2 } }
+								/>
```

Comment:
maybe move the `showBanner` & `markBannerAsViewed` inside the component itself? (just call the hook inside it)

EDIT:
Ok, it requires parameters about the usage. Maybe add a context like in the images?

Reply:
Done.